stories/useInput.story.tsx

```diff
@@ -6,12 +6,19 @@ import Validator from "./util/Validator";
 
 const Demo = () => {
   const LIMIT = 10;
-  const name = useInput("initialName", [
-    {
-      validator: Validator.underLengthString(LIMIT),
-      errorText: `type under ${LIMIT}`
-    }
-  ]);
+  const name = useInput(
+    "",
+    [
+      {
+        validator: Validator.underLengthString(LIMIT),
+        errorText: `type under ${LIMIT}`,
+        option: {
+          hard: true
+        }
+      }
+    ],
+    10
+  );
```

Comment:
별거 아닌데 여기 이 상수가 LIMIT이 맞죵?

```suggestion
  const name = useInput(
    "",
    [
      {
        validator: Validator.underLengthString(LIMIT),
        errorText: `type under ${LIMIT}`,
        option: {
          hard: true
        }
      }
    ],
    LIMIT
  );
```

Reply:
아 그쵸 ㅋㅋ큐ㅠ 맞네요 이 예시에서는 LIMIT과 동일합니다

Reply:
[fix: re-use constants value](https://github.com/bigpicture-kr/default-hooks/pull/10/commits/72dd1eb5630bde577437e5e5211c6cc8e0be05ed)